Lab 1/README.md

```diff
@@ -23,7 +33,7 @@ To push an image:
 
    ```bx login```
    
-   To specify an IBM Cloud region, include the API endpoint.
+   To specify an IBM Cloud region, include the API endpoint. <!-- what does this mean? can we add an example? -->
```

Comment:
We should have an example of the flag or whatever necessary to choose the endpoint or remove this line.

As an FYI, html comments don't show up in the final output if we want to leave some kind of more detailed notes that don't show up:

<img width="963" alt="screen shot 2018-02-07 at 3 25 38 pm" src="https://user-images.githubusercontent.com/13337345/35947151-47b500a6-0c1b-11e8-8359-c93d9377be7d.png">